submissions/crealime/dom-api/js/main.js

```diff
@@ -0,0 +1,149 @@
+document.addEventListener("DOMContentLoaded", function() {
+
+  const url = 'json/movies.json' // Json file with movies data
+  let moviesData = [] // Storage for movies data
+  const menuUl = document.querySelector('.menu__ul')
+  const menu = document.querySelector('.menu')
+  const movieContent = document.querySelector('.main')
+  const movieRandomize = document.querySelector('.menu__random')
+  const menuShowButton = document.querySelector('.hamburger')
+
+  // Fill menu template with data
+  function menuTemplate({id, title}) {
+    return `
+			<div class="menu__li">
+				<a class="menu__link" href="#" title="Seven Random Marvel Movies" data-id="${id}">${title}</a>
+			</div>
+		`
+  }
+
+  // Fill movie template with data
+  function movieTemplate({title, description, release, box, runtime, picture}) {
+
+    return `
+		<section class="movie">
+			<div class="movie__left">
+				<div class="preloader">
+					<div class="spinner"></div>
+				</div>
+				<img class="movie__img" src="${picture}" alt="Captain America: The First Avenger">
+			</div>
+			<div class="movie__center">
+				<h2 class="movie__title">${title}</h2>
+				<div class="movie__info-box">
+					<div class="movie__info">
+						<div class="movie__param">Release Date:</div>
+						<div class="movie__param-value">${release}</div>
+					</div>
+					<div class="movie__info">
+						<div class="movie__param">Box Office:</div>
+						<div class="movie__param-value">${box}</div>
+					</div>
+					<div class="movie__info">
+						<div class="movie__param">Runtime:</div>
+						<div class="movie__param-value">${runtime}</div>
+					</div>
+				</div>
+				<p class="movie__description">${description}</p>
+			</div>
+			<div class="movie__right"></div>
+		</section>
+		`
+  }
+
+  function addActiveClassToMenuItem(id) {
+    const menuLinks = document.querySelectorAll('.menu__link')
+    menuLinks.forEach(el => {
+      el.classList.remove('active')
+      if (el.dataset.id === id) el.classList.add('active')
+    })
+  }
+
+  // Find selected movie and paste it into the main
+  function renderActualMovie(movies, id = '01') {
+
+    movieContent.innerHTML = ''
+
+    const activeMovie = movies.find(el => el.id === id)
+    const activeMovieHTML = movieTemplate(activeMovie)
+
+    movieContent.insertAdjacentHTML('afterbegin', activeMovieHTML)
+
+    addActiveClassToMenuItem(id)
+  }
+
+  // Loop through random movies and paste them into the menu
+  function renderMoviesMenu(movies) {
```

Comment:
Your function has name `renderMoviesMenu` but it do couple things: 
1. renders menu
2. renders first movie (becuase you invoke `renderActualMovie` )

try split this logic

Reply:
Done. Moved the call to the parent function randomMovies